selfdrive/statsd.py

```diff
@@ -104,8 +116,22 @@ def get_influxdb_line(measurement: str, value: float, timestamp: datetime, tags:
       for gauge_key in gauges:
         result += get_influxdb_line(f"gauge.{gauge_key}", gauges[gauge_key], current_time, tags)
```

Comment:
Should we eliminate the metric type prefix?  Using datadog for example I can't think of a time where I wished I had this.  The fields on the metric make it obvious I guess.

Reply:
yeah, I'm open to that. Would have to think about all the metrics we might want to implement in the future (counters, ...), but in any case it's usually indeed pretty obvious or easily looked up in the code if needed